redash/handlers/authentication.py

```diff
@@ -31,6 +31,14 @@ def get_google_auth_url(next_path):
     return google_auth_url
 
 
+def get_github_auth_url(next_path):
+     if settings.MULTI_ORG:
+         github_auth_url = url_for('github_oauth.authorize_org', next=next_path, org_slug=current_org.slug)
+     else:
+         github_auth_url = url_for('github_oauth.authorize', next=next_path)
+     return github_auth_url
```

Comment:
Since both parts of the conditional just end up returning `url_for`, how about:
```suggestion
     kwargs = {"org_slug": current_org.slug} if settings.MULTI_ORG else {}
     return url_for('github_oauth.authorize_org', next=next_path, **kwargs)
```

Reply:
also first argument for `url_for()` is different -> maybe no need to change this part.. Note that google part is also very similar - maybe rather some integration with google part.. hmm